rattler/templates/profile/me.html

```diff
@@ -1 +1 @@
-{% include 'profile/profile.html' %}
+{% include 'profile/profile.html' with profile=user %}
```

Comment:
Vllt. etwas verwirrend, da wir ja in der DB zwischen `user` und `profile` unterscheiden. Und hier wird `user` als `profile` bezeichnet

Reply:
@12jr Da hast du recht. Ich würde es hier an der Stelle vlt. umdrehen. Aktuell gehen wir über den User auf das Profiel. Lass uns vlt. vom Profil auf den User.